packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -26,7 +26,11 @@ export function getBeaconPoolApi({
   return {
     async getPoolAttestations({slot, committeeIndex}) {
       // Already filtered by slot
-      let attestations = chain.aggregatedAttestationPool.getAll(slot);
+      let attestations: Attestation[] = chain.aggregatedAttestationPool.getAll(slot);
+
+      if (attestations.some((att) => isElectraAttestation(att))) {
+        throw new Error(`Aggregated attestation pool contains electra attestations at slot ${slot}`);
```

Comment:
I was rather thinking of an error to indicate that this api is no longer usable and v2 should be used instead.

Reply:
> I was rather thinking of an error to indicate that this api is no longer usable and v2 should be used instead.

Hmmm, I was thinking after Electra is live, we do a release to deprecate all attestation V1 endpoints.

Reply:
Yes we should do that, this comment was rather in regard to the error message, a user querying this api post-electra should be informed to use the v2 api instead.

e.g. an error like this
```ts
throw new ApiError(400, `Use getPoolAttestationsV2 to retrieve electra+ pool attestations fork=${fork}`);
```

could print either fork or slot